atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/transaction/impl/SnapshotTransaction.java

```diff
@@ -523,8 +525,8 @@ private void finishConsumingPreviousRow(PeekingIterator<Map.Entry<Cell, Value>>
                 }
                 if (numConsumed > 0) {
                     log.warn("Not all columns for row {} were read. {} columns were discarded.",
-                             Arrays.toString(prevRowName),
-                             numConsumed);
+                             UnsafeArg.of("row", Arrays.toString(prevRowName)),
+                             SafeArg.of("numColumnDiscarded", numConsumed));
```

Comment:
`numColumnsDiscarded` would be cleaner. Also "consumed" is a bit of a weird word for "discarded" in this whole method.